### PR TITLE
Migrate WPAcceptance tests to Cypress

### PR DESCRIPTION
**Pull request by cadic**
#### DistributedPostTest.php
- [ ] testDistributedCount
- [ ] testDistributedFrom
#### EditorBlocksTest.php
- [ ] addContentToTestPost
- [ ] testBlocksNetworkPushedContent
- [ ] testBlocksNetworkPulledContent
- [ ] testBlocksExternalPushedContent
- [ ] testBlocksExternalPulledContent
#### ExternalConnectionCreateTest.php --> external-connection.test.js
- [x] testCreateExternalConnection
#### InternalPullTest.php --> internal-pull.test.js
- [x] testPostShowingPerStatus
- [x] testPullPost
- [x] testSkipPost
- [x] testSearchMultipleWordsDuringPull
#### InternalPushTest.php --> internal-push.test.js
- [x] testPushDraftPost
- [x] testPushPublishPost
- [x] testPushDataSync
#### LinkUnlinkTest.php
- [ ] testUnlinkPublishPost
#### OembedTest.php
- [ ] testOembedNetworkPushedContent
- [ ] testOembedNetworkPulledContent
- [ ] testOembedExternalPushedContent
- [ ] testOembedExternalPulledContent
#### PushMenuTest.php
- [ ] testMenuItemHover
- [ ] testConnectionCrossOutOnPush
- [ ] testSelectConnection
#### SettingsTest.php --> settings.test.js
- [x] testSettingsSave
- [x] testAuthorBylineSetting

### Changelog Entry

> Changed - Migrate WPAcceptance tests to Cypress

### Credits

Props @cadic

### Checklist:

- [ ] I agree to follow this project's [**Code of Conduct**](https://github.com/10up/.github/blob/trunk/CODE_OF_CONDUCT.md).
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my change.
- [ ] All new and existing tests pass.

### CODE REVIEW

File: .github/workflows/cypress.yml [hunk 1]
```diff
@@ -49,9 +49,6 @@ jobs:
         npx wp-env run cli "wp core version"
         npx wp-env run cli "php --version"
 
-    - name: Convert to multisite
```
Comment (cadic) [inline .github/workflows/cypress.yml:60]: See `tests/bin/initialize.sh`

**Review by Sidsector9 — DISMISSED** [review]
Good work on this @cadic! LGTM 👍

**Comment by cadic** [comment]
@Sidsector9 this is an unfinished work, still needs some tests to be migrated

**Comment by jeffpaul** [comment]
@peterwilsoncc FYI on this in-progress work here that we'll want to ensure gets wrapped before v2 is released so we can ensure these run & validate the refactoring work you're leading

**Comment by peterwilsoncc** [comment]
I'll resolve the merge conflicts on this and merge the progress if everything is working as expected.

**Comment by peterwilsoncc** [comment]
Getting mixed results with the tests, some versions of WP pass, others fail.

I suspect there are some markup changes that will need to be accounted for.

**Comment by jeffpaul** [comment]
Note that #1038 documents the additional tests needed that otherwise were not added in this PR

**Comment by peterwilsoncc** [comment]
> Getting mixed results with the tests, some versions of WP pass, others fail.

This should be resolved by #1056 for #1055.

**Comment by peterwilsoncc** [comment]
The existing tests on this branch are now passing so I will merge this in to restore what we can.

**Review by peterwilsoncc — APPROVED** [review]

**Pull request merged**